Heyitsquoracom/microsoft-graph-explorer-v4
# Create Hey-c6fc3

https://graph.microsoft.com/v1.0/%7Bresource%7D?%5Bquery_parameters%5D


## Overview

Brief description of what this PR does, and why it is needed.

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

* How to test this PR
* Prefer bulleted description
* Start after checking out this branch
* Include any setup required, such as bundling scripts, restarting services, etc.
* Include test case, and expected outputy
